test/common/stats/stats_impl_test.cc

```diff
@@ -170,7 +158,32 @@ class DefaultTagRegexTester {
                        stat_name);
   }
 
-  std::vector<TagExtractorPtr> tag_extractors_;
+  /**
+   * Reimplements TagProducerImpl::produceTags, but extracts the tags in reverse order.
+   * His helps demonstrate that the order extractors does not matter to the end result,
```

Comment:
typo: His -> This, and "order of extractors"

Reply:
done